pkg/inputs/snmp/mibs/profile.go

```diff
@@ -777,7 +782,7 @@ func prune(mibs map[string]*kt.Mib, counterTimeSec int, minCounterTime int) {
 
 	// Lastly, delete if we're not keeping.
 	for oid, mib := range mibs {
-		if !keepNames[oid] {
+		if !keepNames[oid] && !mib.AllowDup {
```

Comment:
> Only metrics which return a value when polling will be used.

Question: it looks like the `allow_duplicate` field is just acting as an override so that the oid is kept as a metric to be polled. But I'm not noticing if the the metric was polled to make sure a value returned. If an OID is polled and nothing is returned, is that just omitted when the metrics get sent?

Reply:
In theory, what happens is first all of the oid get polled: https://github.com/kentik/ktranslate/blob/main/pkg/inputs/snmp/metrics/device_metrics.go#L93

Then, for those that did return a result, we walk over the results and do stuff with them:
https://github.com/kentik/ktranslate/blob/main/pkg/inputs/snmp/metrics/device_metrics.go#L146

What I'm hoping happens here is that this OID will get polled, but since nothing is returned, it will just get dropped in the 2nd loop and not sent back home.